tests/App/AppKernel.php

```diff
@@ -91,6 +96,9 @@ protected function configureRoutes($routes): void
         $routes->import($this->getProjectDir().'/config/routes.yaml');
     }
 
+    /**
+     * @psalm-suppress DeprecatedClass
```

Comment:
A comment about it ?

Reply:
We didn't add it on other bundles. This method is rather simple, it will be removed once SonataBlock bundle is version 5 only.